src/ui_components/search_input.js

```diff
@@ -35,6 +43,7 @@ export default class SearchInput {
 
   handleKeyboard() {
     document.onkeydown = function(e) {
+      if(MAPBOX_RESERVED_KEYS.find((key) => key === e.keyCode)) {console.log( e.keyCode); return}
```

Comment:
Shouldn't we use `e.key` instead of deprecated `e.keyCode` ?
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent#Properties

Reply:
i'll never know, for now i think key is more safe, maybe i'll build an abstraction one day
(key will need a litle polyfill)